### PR TITLE
fix: SV quality filter

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

Fixed: `bfcftools` filter command in SV quality filter to not filter out variants that are filtered out as `MaxDepth`.

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+[X.X.X]
```
Comment (ivadym) [inline CHANGELOG.rst:1]:
```suggestion
[10.0.2]
```

**Review by ivadym — APPROVED** [review]
🧯 👍

File: CHANGELOG.rst (hunk 1 above)
Comment (khurrammaqbool) [reply CHANGELOG.rst:1]: [11.0.2] 🤣 We need rest 🤣

Comment (ivadym) [reply CHANGELOG.rst:1]: 🤣

**Pull request merged**